job_spec/ARIA_S1_GUNW.yml

```diff
@@ -0,0 +1,86 @@
+ARIA_S1_GUNW:
+  required_parameters:
+    - granules
+    - secondary_granules
+    - frame_id
+  parameters:
+    granules:
+      api_schema:
+        type: array
+        minItems: 2
+        maxItems: 3
+        items:
+          description: The name of the Sentinel-1 SLC granules to use as reference scenes for InSAR processing
+          type: string
+          pattern: "^S1[AB]_IW_SLC__1S[SD]V"
+          minLength: 67
+          maxLength: 67
+          example: S1A_IW_SLC__1SDV_20180206T141345_20180206T141414_020487_023099_1F41
+    secondary_granules:
+      api_schema:
+        type: array
+        minItems: 2
+        maxItems: 3
+        items:
+          description: The name of the Sentinel-1 SLC granules to use as secondary scenes for InSAR processing
+          type: string
+          pattern: "^S1[AB]_IW_SLC__1S[SD]V"
+          minLength: 67
+          maxLength: 67
+          example: S1A_IW_SLC__1SDV_20180101T141346_20180101T141415_019962_021FEE_FB0A
+    frame_id:
+      api_schema:
+        description: Subset GUNW products to this frame.
+        type: integer
+        minimum: 0
+        maximum: 27397
+        example: 17882
+  cost_profiles:
+    EDC:
+      cost: 15.0
+    DEFAULT:
+      cost: 1.0
+  validators: []
+  steps:
+    - name: ''
+      image: ghcr.io/access-cloud-based-insar/dockerizedtopsapp
+      command:
+        - ++omp-num-threads
+        - '2'  # 8 vCPUs per 16 GB RAM for the C instance family; 4 for M; 2 for R
+        - --bucket
+        - '!Ref Bucket'
+        - --bucket-prefix
+        - Ref::bucket_prefix
+        - --reference-scenes
+        - Ref::granules
+        - --secondary-scenes
+        - Ref::secondary_granules
+        - --frame-id
+        - Ref::frame_id
+      timeout: 21600
+      compute_environment: Default
+      vcpu: 1
+      memory: 15500
+      secrets:
+        - EARTHDATA_USERNAME
+        - EARTHDATA_PASSWORD
+        - ESA_USERNAME
+        - ESA_PASSWORD
```

Comment:
Is this because DockerizedTopsApp still fetches orbit files from CDSE?

Reply:
Yes, these secrets are used to download orbit files at https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/blob/dev/isce2_topsapp/localize_orbits.py#L23